github-orgs/kubeflow/org.yaml

```diff
@@ -712,208 +712,13 @@ orgs:
-            - sarahmaddox
-            - sasha-gitg
-            - saurabh24292
-            - sen-sam
-            - texasmichelle
-            - vishh
-            - yixinshi
-            - zabbasi
-            - zhenghuiwang
-            - ziamsyed
-            - zichuan-scott-xu
-            - zijianjoy
-            - zpChris
-            - zzmao
-            privacy: closed
           google-admins:
             description: Googlers to ping for help regarding Kubeflow's GitHub org and Google Groups
             members:
             - chensun
             - james-jwu
             - IronPan
             privacy: closed
-          Intel:
-            description: Team of members from Intel
-            members:
-            - ashahba
-            - kkasravi
-            - nqn
-            - balajismaniam
-            privacy: closed
-            repos:
-              pytorch-operator: admin
-          IBM:
-            description: Team of members from IBM
-            members:
-            - ckadner
-            - JosepSampe
-            - revit13
-            - roytman
-            - Tomcli
-            - yhwang
-            privacy: closed
-            repos:
-              kfp-tekton: write
-          Nutanix:
-            description: Team of members from Nutanix
-            members:
-            - johnugeorge
-            privacy: closed
-          Red Hat:
-            description: Team of members from Red Hat
-            members:
-            - adysenrothman
-            - Al-Pragliola
-            - alexcreasy
-            - andyatmiami
-            - anishasthana
-            - asaadbalum
-            - astefanutti
-            - caponetto
-            - Crazyglue
-            - christianvogt
-            - dandawg
-            - dbasunag
-            - DharmitD
-            - dhirajsb
-            - diegolovison
-            - ederign
-            - erikerlandson
-            - fege
-            - Griffin-Sullivan
-            - gmfrasca
-            - gregsheremeta
-            - hbelmiro
-            - HumairAK
-            - isinyaaa
-            - jenny-s51
-            - jiridanek
-            - liavweiss
-            - lucferbux
-            - lugi0
-            - paulovmr
-            - rareddy
-            - rimolive
-            - syntaxsdev
-            - tarilabs
-            - terrytangyuan
-            - thaorell
-            - yehudit1987
-            - YosiElias
-            privacy: closed
-          SUSE:
-            description: Team of members from SUSE
-            members:
-            - alfsuse
-            - gsantomaggio
-            privacy: closed
-          Seldon:
-            description: Team of members from Seldon
-            members:
-            - axsaucedo
-            - gsunner
-            - Maximophone
-            - ryandawsonuk
-            - SachinVarghese
-            privacy: closed
-            repos:
-              example-seldon: write
           blog-admin-team:
```

Comment:
@anishasthana I think, there are more teams that currently unused (e.g. blog-admin-team,  caffe2-team, etc.).
Do you want to cleanup them as part of this PR ?

Reply:
I can add those 2 in, and we can raise more PRs in the future as we determine more unused teams.